src/rules/constructorArgumentsCopy.ts

```diff
@@ -0,0 +1,67 @@
+const goodCode = `
+pragma solidity 0.4.4;
+
+
+contract C {
+    constructor(uint a_);
+}
+`;
+const badCode = `
+pragma solidity 0.4.4;
+
+
+contract C {
+    constructor(uint a);
+}
+`;
+const META = {
```

Comment:
If the rest already works, I suggest to create basic types for those exported things to actually start making use of typescript

Reply:
let's iterate over the rest for now before introducing types.